lib/SILOptimizer/Mandatory/Differentiation.cpp

```diff
@@ -1737,6 +1736,7 @@ static void dumpActivityInfo(SILFunction &fn,
     for (auto &inst : bb)
       for (auto res : inst.getResults())
         dumpActivityInfo(res, indices, activityInfo, s);
+    s << "\n";
```

Comment:
```suggestion
    s << '\n';
```

Reply:
Done in https://github.com/apple/swift/pull/25558/commits/32f66294d96791f67f15234948b493b76b9d31da.